source/agora/consensus/protocol/Nominator.d

```diff
@@ -183,7 +183,6 @@ extern(D):
     public void setQuorumConfig (const ref QuorumConfig quorum,
         const(QuorumConfig)[] other_quorums) nothrow @safe
     {
-        assert(!this.is_nominating);
```

Comment:
This was a useful assert to catch bugs.

Even though SCP by design allows quorums to change mid-nomination, in our design the quorums should be fixed within every new block height nomination.

Reply:
I have added `is_nominating` back for this.